pkgs/tools/wayland/chayang/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, meson
+, ninja
+, pkg-config
+, wayland-scanner
+, wayland-protocols
+, wayland
+}:
+
+stdenv.mkDerivation rec {
+  pname = "chayang";
+  version = "0.1.0";
+
+  src = fetchFromSourcehut {
+    owner = "~emersion";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "3Vu9/Bu2WQe2Yx/2BK25pEpuPNwX6g3qoFUMznCFHeI=";
+  };
+
+  strictDeps = true;
+  depsBuildBuild = [
+    pkg-config
+  ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland-scanner ];
+  buildInputs = [ wayland-protocols wayland ];
```

Inconsistent list style, and these tend to be abit more spaced. Suggestion:

```suggestion
  strictDeps = true;

  depsBuildBuild = [
    pkg-config
  ];

  nativeBuildInputs = [
    meson
    ninja
    pkg-config
    wayland-scanner
  ];
  
  buildInputs = [
    wayland-protocols
    wayland
  ];
```